ArduinoUno.md

```diff
@@ -0,0 +1,228 @@
+## Arduino Uno 
+
+The Arduino Uno connector is a standardised connector in Mbed, which has a set amount of exposed functionality. To achieve meaningful portability of application code across various Mbed enabled boards that are Arduino Uno compliant, the pin names used for the connector pins are common across these boards.
+
+The following diagram shows the Arduino Uno standard for Mbed boards:
+![Static pinmap model](https://raw.githubusercontent.com/ARMmbed/mbed-os/feature-std-pin-name/docs/design-documents/hal/img/Arduino_Uno_Mbed.png) 
+
+**Digital and Analog pins**
+
+The Arduino Uno connector for Mbed boards supports both digital and analog pins. Please note that the Arduino Uno only supports analog inputs and does not support analog output functionality. Digital pin functionality such as GPIO output or input on the Arduino Uno header can be accessed from any of the pins labelled as ARDUINO_UNO_D0 to ARDUINO_UNO_D15 and ARDUINO_UNO_A0 to ARDUINO_UNO_A5. The analog input functionality on the Arduino Uno connector can only be accessed on pins ARDUINO_UNO_A0 to ARDUINO_UNO_A5. The full digital and analog pin alias list for the Arduino Uno connector can be seen below.  
+
+    ARDUINO_UNO_D0 
+    ARDUINO_UNO_D1 
+    ARDUINO_UNO_D2 
+    ARDUINO_UNO_D3 
+    ARDUINO_UNO_D4 
+    ARDUINO_UNO_D5 
+    ARDUINO_UNO_D6 
+    ARDUINO_UNO_D7 
+    ARDUINO_UNO_D8 
+    ARDUINO_UNO_D9 
+    ARDUINO_UNO_D10 
+    ARDUINO_UNO_D11 
+    ARDUINO_UNO_D12 
+    ARDUINO_UNO_D13 
+    ARDUINO_UNO_D14 
+    ARDUINO_UNO_D15 
+    
+    ARDUINO_UNO_A0 
+    ARDUINO_UNO_A1 
+    ARDUINO_UNO_A2 
+    ARDUINO_UNO_A3 
+    ARDUINO_UNO_A4 
+    ARDUINO_UNO_A5 
+    
+**Arduino Uno digital out example**
+
+A simple example on how to create an Arduino Uno digital out pin has been included below for further information on digital pins. For more details about digital functionality, please check the Mbed API.
+     
+    /*
```

Comment:
Note to myself: Need to consider providing this code as snipped examples (to align with others in the docs).